clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoMetalake.java

```diff
@@ -762,6 +771,55 @@ public Group revokeRolesFromGroup(List<String> roles, String group)
     return resp.getGroup();
   }
 
+  /**
+   * Get the owner of a metadata object.
+   *
+   * @param object The metadata object
+   * @return The owner of the metadata object. If the metadata object doesn't have the owner, it
+   *     will return Optional.empty().
+   * @throws NotFoundException If the metadata object or the owner isn't found.
```

Comment:
Since the result type is `Optional`, why do we throw `NotFoundException` if the owner can't be found?

Reply:
Owner is user 1. But user 1 is deleted.

Reply:
Maybe I can add user/group lock when I called the getOwner method to avoid this issue. WDYT?

Reply:
I remove the comment. Because it will happen rarely. May we should check owners before we delete a user.